### ISSUE TITLE
Material Editor Copy/Paste Buttons

### ISSUE THREAD

**Issue by NiklasHenricson** [o3de/o3de#4747]
**Is your feature request related to a problem? Please describe.**
As a terrain designer you spend a lot of time on tedious repeated tasks such as copying similar objects around. In order to save we're requesting a UI/functionality enhancement for better UX.

**Describe the solution you'd like**
Provide Copy/Paste buttons in UI material editor so that we can copy around properties from one material to the next without manual input. This can be applied specifically to offsets and other special fields that need manual interventions from the end users.

**Comment by NiklasHenricson** [comment]
Any news on this one? It's been a while but there's no official word on when in the pipeline we could hope for this feature to come.

**Comment by gadams3** [comment]
Hello! Could you please explain more about the problem you're having and feature you're looking for? Are you looking for the ability to select and copy individual properties or a group of properties from one material to another? We would need more information to determine if and how this should be surfaced in the UI.

Are you aware of the feature that allows you to create child or derived materials from an existing material? Instead of copying and pasting individual values, you can create a primary material that has all of the common properties configured the way you need them. Then using “save as child” from the “file” menu, you can create child materials from that. Child materials will inherit all of the properties from the parent except for those that you explicitly override. If the parent changes all of the derived materials will inherit those modifications.

In addition to that, the material editor and many of our tools are very scriptable. There should be Python examples in the automated testing project. You could, for example, use a Python script to open a material with values you want to “copy”, read those properties from the material, then open all of the materials you want to apply them to, set those values, and save.

These might be options until we have a better idea of the UI changes that need to happen.

**Comment by NiklasHenricson** [comment]
Hello,

Practically, copy/cut/paste button functionality for the many different attributes/settings across material editor by group so that when you create several materials you get to copy values around to save time.


<img width="334" alt="material-editor" src="https://user-images.githubusercontent.com/5524755/161640978-bd333b8b-9230-4aa4-847e-e1e2311a5439.png">

**Comment by santorac** [comment]
Yeah, we had something similar on the old Material Editor in Lumberyard where you could copy and paste a whole group of settings. Be sure to consider using "child materials" as @gadams3 explained above, but I agree that this Copy/Paste feature would be useful. 

(Although I don't think Cut would be applicable, so I suggest we disregard the "X" in your screenshot).
